pkg/ibmvpc/parser.go

```diff
@@ -614,12 +634,32 @@ func getNACLconfig(rc *datamodel.ResourcesContainerModel,
 }
 
 func getTgwObjects(c *datamodel.ResourcesContainerModel,
-	res map[string]*vpcmodel.VPCConfig) map[string]*TransitGateway {
+	res map[string]*vpcmodel.VPCConfig, resourceGroup string) map[string]*TransitGateway {
```

Comment:
Should we also consider the region (`Location`) of the TGW?

Reply:
@adisos ?

Reply:
yes, should probably filter by Location of TGW if the input region was given.